sample_app/Gemfile

```diff
@@ -7,7 +7,7 @@ gem 'aws-sdk-rails', path: '../'
 # gem 'aws-sessionstore-dynamodb', path: '../../aws-sessionstore-dynamodb-ruby'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 7.0'
+gem 'rails', '~> 7.0.0'
```

Comment:
this sample seemed to be built on rails 7.0, so I followed that.
https://github.com/aws/aws-sdk-rails/blob/940d543cb8bcacbe6a492c0a0d1f22324fef9d15/sample_app/config/application.rb#L12
I'll update it another time.